ProcessLib/HydroMechanics/HydroMechanicsFEM.h

```diff
@@ -298,6 +298,8 @@ class HydroMechanicsLocalAssembler
         SpatialPosition x_position;
         x_position.setElementID(_element.getID());
 
+        auto const b_alpha = _process_data.biot_coefficient(t, x_position)[0];
```

Comment:
The Biot-coefficient should remain, as any other material parameter, locally assignable ..